packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -66,7 +66,7 @@ const AppConfig = ({children, locals = {}}) => {
             headers={headers}
             // Uncomment 'enablePWAKitPrivateClient' to use SLAS private client login flows.
             // Make sure to also enable useSLASPrivateClient in ssr.js when enabling this setting.
-            // enablePWAKitPrivateClient={true}
+            enablePWAKitPrivateClient={true}
```

Comment:
Doing this will make private client the default for our local development environment.

I think we want to do this, but we'll need to coordinate with our team on setting the client secret in our environments

Reply:
sorry, this is for me doing local testing. will revert